panel/src/components/Views/ModelView.vue

```diff
@@ -9,6 +9,7 @@ export default {
 		api: String,
 		blueprint: String,
 		buttons: Array,
+		changesUrl: String,
```

Comment:
`Comment:` Not super happy with the name. It doesn't really explain what the URL is for: is that where I send the changes to store (I know it's not). Is something like `changesPreviewUrl` to verbose?

I am wondering if the FormControls could be mostly a `Panel\Ui\Button` like class as well, so that most of it actually gets assigned on the backend. But that's surely not for this PR.

Reply:
I'm not super happy with it either. We still pass the preview URL (for the published version) as part of the model object and I was first tempted to simply use "preview" as a new prop, but then decided against it to avoid confusion. 

I was also thinking about implementing the dropdown on the backend and load it asynchronously. The model view wouldn't need to pass the preview URL in that case at all. 

Maybe we can keep changesURL as a temporary solution, since we both agree that the final implementation for the form controls needs to be different anyway.